docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -30,88 +30,3 @@ This limit will be applied to every container of the given component.
 +
 For `plugin` components, RAM limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-
```

Comment:
Memory request has been reintroduced to devfile 2.1 spec (https://docs.devfile.io/devfile/2.1.0/user-guide/api-reference.html, https://github.com/devfile/api/issues/131). Given that this change is on master which is currently mapping to 2.1 spec, this section should stay.